src/modules/navigator/mission_block.cpp

```diff
@@ -201,10 +201,12 @@ MissionBlock::is_mission_item_reached()
 			}
 
 			/* require only altitude for takeoff for multicopter */
-			if (_navigator->get_global_position()->alt >
-			    altitude_amsl - altitude_acceptance_radius) {
-				_waypoint_position_reached = true;
-			}
+			const auto global_altitude_acceptable =
+				_navigator->get_global_position()->alt > altitude_amsl - altitude_acceptance_radius;
```

Comment:
Check validity before using global?

Reply:
Definitely. I'll rebase and revisit this PR soon.